### DISCUSSION TITLE
Invalid URL & Failed to connect with walletFailed to login with openlogin

### DISCUSSION THREAD

**Discussion by fachrudahri** [wevm/wagmi#1612] [Q&A] [answered]
hello devs, I'm having a problem after updating to wagmi 0.9.2.
the problem I encountered was when logging in using web3auth-wagmi-connector, then get an error invalid URL & Failed to connect with walletFailed to login with openlogin.
I have also checked the documentation, there is no update on web3auth-wagmi-connector.

```
const checkingDeployment = isBrowserURL()  ? bsc : goerli;

const { chains, provider, webSocketProvider } = configureChains(
  [checkingDeployment],
  [
    alchemyProvider({ apiKey: process.env.APIKEY_ALCHEMY, priority: 0 }),
    publicProvider({ priority: 1 })
  ],
);

const client = createClient({
  autoConnect: true,
  connectors: [
    new Web3AuthConnector({
      chains,
      options: {
        whiteLabel: {
          name: 'Nano'
        },
        storageKey: 'local',
        clientId: process.env.CLIENT_ID_WEB3AUTH, // Get your own client id from https://dashboard.web3auth.io
        network: "cyan", // web3auth network, "mainnet", "cyan", or "aqua"
        chainId: "0x5", // chainId that you want to connect with
        uiConfig: {
          theme: "light",
          loginMethodsOrder: ["google"],
          appLogo: `${logo.src}`, // Your App Logo Here
        },
      },
    }),
  ],
  provider,
  webSocketProvider,
});
```

![Screen Shot 2023-01-04 at 13 17 01](https://user-images.githubusercontent.com/42551950/210496349-2cd01623-923c-43ec-a69c-8da87128c2a4.png)

**Comment by tmm** [comment, marked as answer]
Can you post a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example)?

**Reply by fachrudahri** [reply]
sorry I didn't include it before, I've added the code in this post to setup web3auth-wagmi-connector.
Thank you for seeing this post, I hope this problem can be resolved.

**Reply by fachrudahri** [reply]
is there a solution? @tmm

**Reply by tmm** [reply]
The issue seems related to `Web3AuthConnector`, which is not an official wagmi Connector so not sure how it works.
